subsys/net/lib/nrf_cloud/src/nrf_cloud.c

```diff
@@ -408,44 +462,149 @@ static int send(const struct cloud_backend *const backend,
 	return 0;
 }
 
-static int ping(const struct cloud_backend *const backend)
+static int api_ping(const struct cloud_backend *const backend)
 {
 	/* TODO: Do only ping, nrf_cloud_process() also checks for input. */
 	nrf_cloud_process();
 
 	return 0;
 }
 
-static int keepalive_time_left(const struct cloud_backend *const backend)
+static int api_keepalive_time_left(const struct cloud_backend *const backend)
 {
 	return nct_keepalive_time_left();
 }
 
-static int input(const struct cloud_backend *const backend)
+static int api_input(const struct cloud_backend *const backend)
 {
 	nrf_cloud_process();
 
 	return 0;
 }
 
-static int user_data_set(const struct cloud_backend *const backend,
+static int api_user_data_set(const struct cloud_backend *const backend,
 			 void *user_data)
 {
 	backend->config->user_data = user_data;
 
 	return 0;
 }
 
+void connection_poll(void)
```

Comment:
Pobabably my nitpicking, but I'm not in favour of `connection_poll` name, I was actually quite surprised to see that it's an entry point for a thread. Since this function becomes a core for the nrf_cloud processing (if enabled), how about calling it `nrf_cloud_process` or `nrf_cloud_run`?

Reply:
sounds good